src/backend/core/api/viewsets/label.py

```diff
@@ -232,8 +235,17 @@ def create(self, request, *args, **kwargs):
             models.Label._meta.get_field("color").default,  # noqa: SLF001
         )
 
+        description = serializer.validated_data.get("description", "")
+        is_auto = serializer.validated_data.get("is_auto", True)
```

Comment:
IMO you should not set default values. Default values from the fields model should be used instead.
```suggestion
        description = serializer.validated_data.get("description")
        is_auto = serializer.validated_data.get("is_auto")
```